gdal/frmts/makefile.vc

```diff
@@ -15,7 +15,7 @@ EXTRAFLAGS =	-DFRMT_ceos -DFRMT_aigrid -DFRMT_elas -DFRMT_hfa -DFRMT_gtiff\
 		-DFRMT_r -DFRMT_northwood -DFRMT_saga -DFRMT_xyz -DFRMT_hf2 \
 		-DFRMT_kmlsuperoverlay -DFRMT_ozi -DFRMT_ctg -DFRMT_e00grid \
 		-DFRMT_zmap -DFRMT_ngsgeoid -DFRMT_iris -DFRMT_map -DFRMT_cals \
-		-DFRMT_safe -DFRMT_sentinel2 -DFRMT_derived -DFRMT_prf
+		-DFRMT_safe -DFRMT_sentinel2 -DFRMT_derived -DFRMT_prf -DFRMT_jpegxr
```

Comment:
Should only be defined if JPEGXR support is enabled in nmake.opt or its override file.